clients/fides-js/.eslintignore

```diff
@@ -1,6 +1,6 @@
 node_modules
 dist
-jest.config.js
+**/*.js
```

Comment:
Seems a *bit* broad... 😁

Reply:
eh, I'll double check it. I think eslint is always going to have a problem with .js files.